run-automation-test.md

```diff
@@ -0,0 +1,145 @@
+# Running Automation test with Kobiton
+## Table of contents
++ [Prerequisites](#prerequisites)
++ [1. Configure TeamCity and integrate with GitHub](#1-configure-teamcity-and-integrate-with-github)
++ [2. Configure automation test script](#2-configure-automation-test-script)
++ [3. Run automation test on Kobiton devices](#3-run-automation-test-on-kobiton-devices)
++ [4. Fetch test session data through Kobiton REST API](#4-fetch-test-session-data-through-kobiton-rest-api)
++ [5. Feedback](#5-feedback)
+## Prerequisites 
+- Java(JRE). Supported are:
+  + Oracle java 8 or higher
+  + OpenJDK 8
+- Kobiton account
+  >Please visit https://portal.kobiton.com/register to create new account.
+- Node: `6.0.0.` or higher
+## 1. Configure TeamCity and integrate with GitHub
+Firstly, let's assume you already have an empty GitHub repository for running automation test.
+
+This part will guide you to how to configure TeamCity to integrate with your automation test GitHub repository. Skip this step if you have already had TeamCity setup.
+ + For instruction on how to install and configure TeamCity server, follow [this guide](https://confluence.jetbrains.com/display/TCD18/Installing+and+Configuring+the+TeamCity+Server).
+
+Set up your project and link your app repository that we will test with Kobiton later.
++ You need to configure a connection to your GitHub repository. Instruction on how to connect GitHub with TeamCity can be found [this guide](https://confluence.jetbrains.com/display/TCD10/Integrating+TeamCity+with+VCS+Hosting+Services), in `Connecting to GitHub` section. The connection is configured, and now TeamCity will be granted full control of private repositories and the Write repository hooks permission.
+ 
+>For more information about TeamCity can be found in [this tutorial](https://confluence.jetbrains.com/display/TCD10/TeamCity+Documentation).
+
+## 2. Configure automation test script
+### 2.1 Get Username and API key
+Go to https://portal.kobiton.com and login to your Kobiton account.
+- Username
+  + Click *user icon* -> **"Profile"** (you might find it in the top right corner)
+
+  ![get username](./assets/username.png )
```

Comment:
This image is inconsistent with: https://github.com/kobiton/Travis-HockeyApp-Appium-ReactNative/pull/4/files#diff-ada87b81f92676697c82db599c21d8acR42

Please change this image. Apply for others.

Reply:
Agree, @phanthaoIT